firmware/thruna/DCM/Src/App/states/App_DriveState.c

```diff
@@ -5,8 +5,10 @@
 #include "states/App_InitState.h"
 #include "App_SetPeriodicCanSignals.h"
 #include "App_SharedConstants.h"
+#include "../../../Inc/App/App_Regen.h"
```

Comment:
App_Regen.h is ok. The header files are all linked in CMake.

Reply:
what do you mean by this? I had issues if I didn't include it so if theres a way i dont need to include it that would be great to know actually.